components/arm/arm.go

```diff
@@ -72,17 +72,55 @@ type Arm interface {
 	referenceframe.InputEnabled
 
 	// EndPosition returns the current position of the arm.
+	//
+	//    myArm, err := arm.FromRobot(machine, "my_arm")
+	//    // Get the end position of the arm as a Pose.
+	//    pos, err := myArm.EndPosition(context.Background(), nil)
 	EndPosition(ctx context.Context, extra map[string]interface{}) (spatialmath.Pose, error)
 
 	// MoveToPosition moves the arm to the given absolute position.
 	// This will block until done or a new operation cancels this one
+	//
+	//    myArm, err := arm.FromRobot(machine, "my_arm")
+	//    // Create a Pose for the arm.
+	//    examplePose := spatialmath.NewPose(
+	//            r3.Vector{X: 5, Y: 5, Z: 5},
+	//            &spatialmath.OrientationVectorDegrees{0X: 5, 0Y: 5, Theta: 20}
+	//    )
+	//
+	//    // Move your arm to the Pose.
+	//    err = myArm.MoveToPosition(context.Background(), examplePose, nil)
+	//	  if err!=nil {
+	//        logger.Error(err)
+	//        return
+	//    }
 	MoveToPosition(ctx context.Context, pose spatialmath.Pose, extra map[string]interface{}) error
 
 	// MoveToJointPositions moves the arm's joints to the given positions.
 	// This will block until done or a new operation cancels this one
+	//
+	//    // Assumes you have imported componentpb from "go.viam.com/api/component/arm/v1"
+	//    myArm, err := arm.FromRobot(machine, "my_arm")
+	//
+	//    // Declare an array of values with your desired rotational value for each joint on the arm.
+	//    degrees := []float64{4.0, 5.0, 6.0}
+	//
+	//    // Declare a new JointPositions with these values.
+	//    jointPos := &componentpb.JointPositions{Values: degrees}
+	//
+	//    // Move each joint of the arm to the position these values specify.
+	//    err = myArm.MoveToJointPositions(context.Background(), jointPos, nil)
+	//    if err!=nil {
+	//        logger.Error(err)
+	//        return
+	//    }
 	MoveToJointPositions(ctx context.Context, positionDegs *pb.JointPositions, extra map[string]interface{}) error
 
 	// JointPositions returns the current joint positions of the arm.
+	//    my_arm , err := arm.FromRobot(machine, "my_arm")
```

Comment:
`myArm` and formatting (no space before comma). Always camel case for Go var names 🐫 .